File: modul-2/PRAK110-2210817210012-MuhammadRakaAzwar.py
import math 
a, c= 12, 5
b = math.sqrt((a*a)+(c*c))
print("Diketahui :")
print("Alas =", c)
print("Tinggi =", a)
print(" ")
print("Sisi A = %d cm" % a)
print("Sisi B = %d cm" % round(math.sqrt((a*a)+(c*c))))
print("Sisi C = %d cm" % c)
print("Keliling = %d cm" % (a+b+c))
print("Luas = %d cm" % (0.5*a*c))

File: modul-2/PRAK110-2210817210012-MuhammadRakaAzwar.c
#include <stdio.h>
int main(){
float a = 12, c = 5, b = sqrt((a*a)+(c*c));
printf("Diketahui :\n");
printf("Alas = %.f cm\n", c);
printf("Tinggi = %.f cm\n", a);
printf("\n");
printf("Sisi A = %.f cm\n", a);
printf("Sisi B = %.f cm\n", sqrt((a*a)+(c*c)));
printf("Sisi C = %.f cm\n", c);
printf("Keliling = %.f cm\n", a+b+c);
printf("Luas = %.f cm\n", 0.5*a*c);
}
